pom.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	This is a template for a Reactome Maven based project.
+	Please customize this template by filling in values in square brackets.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.reactome.[optionalsubdomain]</groupId>
+	<artifactId>[lower-case-name-using-kebab-case]</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>[name-of-project]</name>
+	<description>[description of project]</description>
+	<url>https://reactome.org</url>
+	<licenses>
+		<license>
+			<name>[license name]</name>
+			<url>[license url]</url>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<name>[Your Name]</name>
+			<email>[your@workemail.com]</email>
+			<organization>Reactome</organization>
+			<organizationUrl>https://reactome.org</organizationUrl>
+		</developer>
+	</developers>
+
+	<scm>
+		<connection>scm:git:git://github.com/reactome/[name-of-repository].git</connection>
+		<developerConnection>scm:git:ssh://github.com/reactome/[name-of-repository].git</developerConnection>
+		<url>https://github.com/reactome/[name-of-repository]</url>
+		<tag>develop</tag>
+	</scm>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!--
+		integration tests run by default when running the command "mvn verify"; can be overridden using the option
+		"-DskipITs=true"
+		-->
+		<skipITs>false</skipITs>
+
+		<!--
+		make a property for any dependency versions used more than once
+		-->
+		<junit.version>5.5.2</junit.version>
+		<log4j.version>2.13.0</log4j.version>
+	</properties>
+
+	<dependencies>
+		<!--
+		Main dependencies
+			- List dependencies needed by the main code base in this section
+		-->
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-api</artifactId>
+			<version>${log4j.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+			<version>${log4j.version}</version>
+		</dependency>
+
+		<!--
+		Test dependencies
+			- List dependencies needed ONLY by tests in this section
+		-->
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest</artifactId>
+			<version>2.2</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-api</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
```

Comment:
Maybe each of the template dependencies should have a description that explains what they are/why they are needed?

Reply:
Okay - will do!